README.md

```diff
@@ -22,52 +22,36 @@ The Delphix Plugin allows Jenkins to connect to multiple Delphix Engines and exe
 
 ## <a id="description"></a>Description
 
-This plugin is designed to automate routine and/or triggered jobs with Delphix Self Service Containers and Bookmarks. An example automation would be during a build phase, if some automated tests failed while running in a database, that Self Service Container could have it's current state bookmarked, and that bookmark could be shared with the rest of the team. It might also be required to refresh a development database from the production source during deployment. These are just a few examples of possible automations.
+This plugin is designed to automate routine and/or trigger jobs with DCT.
 
 ## <a id="installation"></a>Installation
 
 Install through Jenkins Plugin Manager or download [here](https://plugins.jenkins.io/delphix).
 
 ## <a id="requirements"></a>Delphix Engine Requirements
 
-Delphix Engine v5.0.0 or higher.
+Delphix Engine v6.0.0 or higher.
 
 ## <a id="usage"></a>Usage
 
 #### Global Configuration
 
-After the plugin has been installed, Delphix engines will need to be connected. Go to System Configuration page, scroll down to the Delphix section. Enter your Engine Address, Username, and password, then click "Test Connection". If Jenkins is able to reach the engine and log in, you will see a "Login Successful" reply message. If you wish to add multiple engines, click the "Add Engine" button for each engine you wish to connect to Jenkins.
+After the plugin has been installed, DCT will need to be connected. Go to System Configuration page, scroll down to the Delphix section. Enter your DCT URL.
 
-#### Self Service Container
+#### Credentials
 
-To interact with Self Service Containers, select "Delphix - Self Service Container" from the "Add Build Step" drop down in the Build phase section. Select an engine that was configured in the global configuration, select one of the operations listed below. Depending on the operation selected, it will either depend on a specified Self Service Container or a Bookmark. Read more about [Delphix Self Service Containers](https://docs.delphix.com/docs/delphix-self-service/delphix-self-service-admin-guide/understanding-data-containers).
+The API KEY has to be saved on Jenkins’s side as a Secret text , In Jenkins, go to Credentials > global > Add Credentials, select the Secret text type and fill in the inputs.
```

Comment:
Update Line: 
`The API KEY has to be saved on Jenkins’s side as a Secret text. In Jenkins, go to Credentials > Global > Add Credentials, select the Secret text type, and fill in the inputs.`